File: 4020-elevator-requests-i/4020-elevator-requests-i.py
class Solution(object):
    def elevatorRequests(self, n, requests):
        """
        :type n: int
        :type requests: List[int]
        :rtype: int
        """
        prefix = [0] * len(requests)
        prefix[0] = requests[0]
        for i in range(1 , len(requests)) :
            prefix[i] = abs(requests[i] - requests[i - 1])
        return sum(prefix)

File: 4020-elevator-requests-i/4020-elevator-requests-i.java
class Solution {
    public int elevatorRequests(int n, int[] requests) {
        int time = requests[0]  , prefix = requests[0] ; 

        for (int i = 1 ; i < requests.length ; i++) 
        {
            time += Math.abs(prefix - requests[i]) ; 
            prefix = requests[i] ; 
        }

        return time ; 
    }
}